Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StructRendering.kt

```diff
@@ -62,7 +63,7 @@ private fun tryRenderUnion(def: StructDef): String? =
 
         }
 
-private fun tryRenderVar(type: Type, name: String): String? = when (type) {
+private fun tryRenderVar(type: Type, name: String, shouldOmitConstQualifier: Boolean = false): String? = when (type) {
```

Comment:
Omitting `const` should be default behaviour.
Moreover, I don't know any usage of this function that requires `const` preserved.